mypy.ini

```diff
@@ -0,0 +1,7 @@
+[mypy]
+python_version = 3.6
+namespace_packages = True
+ignore_missing_imports = True
+
+[mypy-google.protobuf]
+ignore_missing_imports = True
```

Comment:
Adding `types-protobuf` to the installed `mypy` environment will make this unnecessary.